build.gradle

```diff
@@ -278,71 +288,67 @@ subprojects {
   // Publishing
   // ----------
 
-  artifacts {
-    archives javadocJar, sourcesJar, shadowJar, testlibJar
-  }
+  if (!nonReleaseProjects.contains(project.name)) {
+    publishing {
+      publications {
+        mavenJava(MavenPublication) {
+          version = project.version
 
-  signing {
-    required { gradle.taskGraph.hasTask("uploadArchives") }
-    sign configurations.archives
-  }
+          from components.java
 
-  if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword') &&
-      !nonReleaseProjects.contains(project.name)) {
-    uploadArchives {
-      repositories {
-        mavenDeployer {
-          beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+          artifact javadocJar
+          artifact sourcesJar
+          artifact shadowJar
+          artifact testlibJar
 
-          repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-            authentication(userName: ossrhUsername, password: ossrhPassword)
-          }
-
-          snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
-            authentication(userName: ossrhUsername, password: ossrhPassword)
-          }
-
-          pom.project {
-            name 'GAX (Google Api eXtensions)'
-            packaging 'jar'
-            artifactId project.name
-            description 'Google Api eXtensions'
-            url 'https://github.com/googleapis'
+          pom {
+            name = 'GAX (Google Api eXtensions)'
```

Comment:
Should it mention Java (since there are many different gax-es?

Reply:
I'm fine with fixing the data in the POM, but preferably in a separate PR